src/sinks/datadog/traces/config.rs

```diff
@@ -54,7 +54,7 @@ impl SinkBatchSettings for DatadogTracesDefaultBatchSettings {
 }
 
 /// Configuration for the `datadog_traces` sink.
-#[configurable_component(sink("datadog_traces"))]
+#[configurable_component(sink("datadog_traces", "Publish traces to Datadog."))]
```

Comment:
Nit, matching the other descriptions.
```suggestion
#[configurable_component(sink("datadog_traces", "Publish trace events to Datadog."))]
```